crates/storage/libmdbx-rs/src/environment.rs

```diff
@@ -767,6 +773,20 @@ impl EnvironmentBuilder {
         self
     }
 
+    /// Controls interprocess/shared threshold to force flush the data buffers to disk, if
+    /// [`SyncMode::SafeNoSync`] is used.
+    pub fn set_sync_bytes(&mut self, v: usize) -> &mut Self {
+        self.sync_bytes = Some(v as u64);
+        self
+    }
+
+    /// Controls interprocess/shared relative period since the last unsteady commit to force flush
+    /// the data buffers to disk, if [`SyncMode::SafeNoSync`] is used.
+    pub fn set_sync_period(&mut self, v: u64) -> &mut Self {
```

Comment:
do we know what value is supposed to be passed here? Would be nice to accept a `Duration` and then convert it into an appropriate `u64`.

Reply:
1/65536 of a second. I can do that, but it will break the direct translation to the C API a bit.
see here.
https://erthink.github.io/libmdbx/group__c__settings.html#ga7e64d98f7a8c3bdf7727627782f1be0a:~:text=%5Bin%5D-,seconds_16dot16,a%20synchronous%20flush%20would%20be%20made%20since%20the%20last%20unsteady%20commit.,-Returns

Reply:
let's do it, because we already do a similar conversion in another part of the codebase https://github.com/paradigmxyz/reth/blob/e83968c79dc21e900e473256fee6b1326f5269da/crates/storage/libmdbx-rs/src/transaction.rs#L681-L690

Reply:
Done